File: Matrix Audio/matrix.audio.client/src/app/models/payment-info.ts
export interface PaymentInfo {
    cardNumber: string;
    cardHolderName: string;
    expirationDate: string;
    cvv: string;
}

export const EmptyPaymentInfo: PaymentInfo = {
    cardNumber: '',
    cardHolderName: '',
    expirationDate: '',
    cvv: ''
};

File: Matrix Audio/Matrix.Audio.Models/PaymentInfo.cs
namespace Matrix.Audio.Models;
public class PaymentInfo
{
    public string CardNumber { get; set; } = string.Empty;
    public string CardHolderName { get; set; } = string.Empty;
    public string ExpirationDate { get; set; } = string.Empty; // Format: MM/YY
    public string Cvv { get; set; } = string.Empty;
}

